gcs/CHANGES.md

```diff
@@ -1,32 +1,32 @@
 ### 3.0.0 - 2022-XX-XX
 
-1.  Remove Hadoop 2.x support.
+1. Remove Hadoop 2.x support.
 
-1.  Update all dependencies to the latest versions.
+2. Update all dependencies to the latest versions.
```

Comment:
I think previous formatting was better (this is a default google3 md format: http://go/mdformat#intellij). It allows, for example, re-arrange release notes without changing their numbers manually.

Reply:
Reverted.